From 1d7c84838445981a06812869f8906bdef52e69eb Mon Sep 17 00:00:00 2001
From: Cristian Stoica <cristian.stoica@nxp.com>
Date: Mon, 15 Feb 2016 18:27:35 +0200
Subject: [PATCH 4/9] fix type of returned value

The function is declared as unsigned int so we return an
unsigned int as well

Signed-off-by: Cristian Stoica <cristian.stoica@nxp.com>
---
 ioctl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ioctl.c b/ioctl.c
index 0385203..db7207a 100644
--- a/ioctl.c
+++ b/ioctl.c
@@ -1065,7 +1065,7 @@ cryptodev_compat_ioctl(struct file *file, unsigned int cmd, unsigned long arg_)
 static unsigned int cryptodev_poll(struct file *file, poll_table *wait)
 {
 	struct crypt_priv *pcr = file->private_data;
-	int ret = 0;
+	unsigned int ret = 0;
 
 	poll_wait(file, &pcr->user_waiter, wait);
 
-- 
2.7.4

